_pkgdown.yml

```
@@ -13,21 +13,28 @@ template:
       google: {family: "Atkinson Hyperlegible", wght: [300, 400, 700], ital: [0, 1]}
     headings-color: "#206B72"
     link-color: "#216B73"
-    link-decoration: underline var(--bs-info) dotted
+    link-color-dark: "#cef4f9"
+    link-decoration: underline dotted
   params:
     footer: <b>bslib</b> is an R package developed by <a href='https://rstudio.com'>RStudio</a>
     docsearch:
       api_key: 53fd87c79a13e431b4298611470d023a
       index_name: bslib
+  includes:
+    in_header: >
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
```

Maybe slightly better to stay current?

```suggestion
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@latest/font/bootstrap-icons.css">
```